src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -98,9 +98,7 @@ export function ChangeDataView({
 
   useEffect(() => {
     const fetchDataViews = async () => {
-      const dataViewsRefs: DataViewListItemEnhanced[] = savedDataViews
```

Comment:
I haven't tested this but why are we removing this?

Reply:
@stratoula now it just contains a smallish refactoring